src/scripts/main.js

```diff
@@ -1,3 +1,25 @@
 'use strict';
 
-// write your code here
+const arrayPopulationStr = [
+  ...document.getElementsByClassName('population'),
+].map((n) => n.textContent);
+
+const arrayPopulationNum = arrayPopulationStr.map((str) => {
+  return Number(
+    str
+      .split('')
+      .filter((element) => element !== ',')
+      .join(''),
+  );
+});
+
+const total = arrayPopulationNum.reduce(
+  (accumulator, currentValue) => accumulator + currentValue,
+  0,
+);
+const average = total / arrayPopulationNum.length;
+const spanTotalPopulation = document.querySelector('.total-population');
+const spanAveragePopulation = document.querySelector('.average-population');
+
+spanTotalPopulation.textContent = total.toLocaleString();
+spanAveragePopulation.textContent = average.toLocaleString();
```

Comment:
The use of toLocaleString() is appropriate here to format the numbers with commas for better readability.